quasar/main.tf

```
@@ -499,3 +499,19 @@ resource "aws_db_instance" "quasar" {
   performance_insights_enabled    = true
   enabled_cloudwatch_logs_exports = ["postgresql", "upgrade"]
 }
+
+
+# Provide S3 Bucket for Customer.io data file exports.
+module "iam_user" {
+  source = "../components/iam_app_user"
+  name   = var.name
```

The build (which can be found by clicking the :x: on the commit) is failing with this error:

```
Error: Reference to undeclared input variable

  on main.tf line 507, in module "iam_user":
 507:   name   = var.name

An input variable with the name "name" has not been declared. This variable
can be declared with a variable "name" {} block.
```

That's because you don't have a [variable](https://www.terraform.io/docs/configuration/locals.html) named `name` in this module (you'd see a `variable {}` block at the top if you did). You could define a [local value](https://www.terraform.io/docs/configuration/locals.html) so you could reference the same "name" for both of these resources, or just provide an inline string here. This will be used as the name of the S3 bucket & user:

```suggestion
  name   = "quasar-cio"
```

---

Woohoo! Setting a local value worked! Thanks @DFurnes!